example/lib/widgets/buy_sheet.dart

```diff
@@ -280,6 +337,36 @@ class BuySheetState extends State<BuySheet> {
     }
   }
 
+  void _onBuyerVerificationSuccess(BuyerVerificationDetails result) async {
+    if (!_chargeServerHostReplaced) {
+      InAppPayments.completeCardEntry(
+        onCardEntryComplete: _onCardEntryComplete);
+
+      _showUrlNotSetAndPrintCurlCommand(result.nonce, verificationToken:result.token);
+      return;
+    }
+
+    try {
+      await chargeCardAfterBuyerVerification(result);
+      InAppPayments.completeCardEntry(
+          onCardEntryComplete: _onCardEntryComplete);
+    } on ChargeException catch (ex) {
+      InAppPayments.showCardNonceProcessingError(ex.errorMessage);
+      showAlertDialog(
+        context: BuySheet.scaffoldKey.currentContext,
+        title: "Error verifying buyer",
```

Comment:
This should be `Error processing card payment` as buyer verification has completed successfully.